src/Core/System/SalesChannel/Context/LanguageInfo.php

```diff
@@ -12,8 +12,16 @@
 #[Package('framework')]
 final readonly class LanguageInfo
 {
+    /**
+     * @internal
```

Comment:
would that be okay?

Reply:
if you have a factory for this, yes. 

And a static function for test cases.

Reply:
How is this not a break? This wasn't internal before